easybuild/easyconfigs/f/FSL/FSL-6.0.1_Fix_fsl_exec_script.patch

```diff
@@ -0,0 +1,11 @@
+diff -Nru fsl.orig/src/misc_tcl/fsl_exec.tcl fsl/src/misc_tcl/fsl_exec.tcl
```

Comment:
@casparvl Filename should be `FSL-6.0.2_*.patch`?

Also, please include a short description of this patch on top (+ mention patch author or upstream reference).

Make sure you update the SHA256 checksums for the patches accordingly.

Reply:
I developed the patch originally for FSL-6.0.1, where it also applies. It fixes an error when running one of the FSL binaries, quite possibly, people simply hadn't encountered it before. I'd propose to check it in like this. If desired, we can include the patch in any FSL-6.0.1 EasyConfig later on...

Reply:
:+1: